src/components/Nav/Nav.Props.ts

```diff
@@ -78,17 +78,21 @@ export interface INavLink {
   isExpanded?: boolean;
 
   /**
-   * Any additional properties to apply to the rendered links.
+   * (Optional) Aria label for nav link
    */
-  [propertyName: string]: any;
+  ariaLabel?: string;
 
   /**
-   * (Optional) Aria label for nav link
+   * (Optional) Unique ID for nav link
    */
-  ariaLabel?: string;
+  id?: string;
 
   /**
-   * (Optional) Title for nav link
+   * (Optional) Desciption
    */
-  title?: string;
+  tooltip?: string;
```

Comment:
this should stay as "title". It is confusing to rename the standard "title" attribute to "tooltip".